radio/src/targets/common/arm/stm32/stm32_pulse_driver.cpp

```diff
@@ -51,19 +53,23 @@ void stm32_pulse_init(const stm32_pulse_timer_t* tim)
 void stm32_pulse_deinit(const stm32_pulse_timer_t* tim)
 {
   // Disable IRQs
-  NVIC_DisableIRQ(tim->DMA_IRQn);
+  if (tim->DMAx) {
+    NVIC_DisableIRQ(tim->DMA_IRQn);
+  }
   NVIC_DisableIRQ(tim->TIM_IRQn);
-  
-  // De-init DMA & timer
-  LL_DMA_DeInit(tim->DMAx, tim->DMA_Stream);
+
+  if (tim->DMAx) {
+    // De-init DMA & timer
+    LL_DMA_DeInit(tim->DMAx, tim->DMA_Stream);
+  }
   LL_TIM_DeInit(tim->TIMx);
 
   // Reconfigure pin as output
```

Comment:
the comment is wrong now and nobody needs it, the code is simple enough

Reply:
the comment is still there
Line 66, "reconfigure pin as output"
Line 71, LL_GPIO_MODE_INPUT